advanced/am-pattern-1/confs/am/instance-1/repository/conf/deployment.toml

```diff
@@ -151,78 +76,16 @@ allow_credentials = false
 enable_data_publishing = false
```

Comment:
@Chathuru data publishing needs to be enabled. Can you please check the default configurations?

Reply:
Fix in https://github.com/Chathuru/kubernetes-apim/commit/0205f6311ce5b9ff81ebe108aa79ff7a040078e1 commit